modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -63,6 +63,7 @@
 		"Bitdomain Technician",
 		"Data Retrieval Specialist",
 		"Netdiver",
+		"Netrunner",
```

Comment:
I've already denied it once, I'll deny it again. That's a title from Cyberpunk, I'm not interested, as much as I love that game.

Reply:
Never even knew that. I just thought it was a neat thing.

Reply:
And, that isn't me playing dumb, I've never played nor have any intent to play Cyberrunner, I just thought Netrunner ran off the tongue well.